spec/fixtures/wikis.yml

```diff
@@ -11,3 +11,13 @@ default:
   id: 1
   language: en
   project: wikipedia
+
+fr:
+  id: 3
+  language: fr
+  project: wikipedia
+
+de:
+  id: 4
+  language: de
+  project: wikipedia
```

Comment:
I would rather use `stub_wiki_validation` in more places, instead of adding additional wikis to the fixtures.

Reply:
I've used these now in course_cloning_spec.rb